src/Symfony/Bundle/WebServerBundle/Command/ServerRunCommand.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebServerBundle\Command;
+
+use Symfony\Bundle\WebServerBundle\WebServer;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Runs Symfony application using a local web server.
+ *
+ * @author Michał Pipa <michal.pipa.xsolve@gmail.com>
+ */
+class ServerRunCommand extends ServerCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDefinition(array(
+                new InputArgument('addressport', InputArgument::OPTIONAL, 'The address to listen to (can be address:port, address, or port)', '127.0.0.1:8000'),
```

Comment:
Minor: instead of `addressport`, we could call this `uri` because we accept schemes or not, IP address or hostnames, port numbers or not, etc.  So this looks like a URI.